spec/helpers/visit_helper_spec.rb

```diff
@@ -60,6 +60,14 @@ def visit
       expect(helper.prison_address).to start_with "1 Fort Road"
     end
 
+    it "provides illegal html in the address and doesn't return illegal html" do
+      allow(Rails.configuration.prison_data['Rochester']).
+        to receive(:[]).with('address') { ['Danger<script>ous', 'apoel'] }
+
+      expect(helper.prison_address).not_to match(/<script/)
+      expect(helper.prison_address).to match(/<br/)
```

Comment:
I think I *would* split these up, but not for single line/multiple line: instead, I'd test *escapes HTML in the prison address* and *joins multiple lines with unescaped glue string*, which would end up with one expectation each.